lxc/remote.go

```diff
@@ -167,72 +165,107 @@ func (c *cmdRemoteAdd) RunToken(server string, token string, rawToken *api.Certi
 	for _, addr := range rawToken.Addresses {
 		addr = fmt.Sprintf("https://%s", addr)
 
-		conf.Remotes[server] = config.Remote{Addr: addr, Protocol: c.flagProtocol, AuthType: c.flagAuthType, Domain: c.flagDomain}
-
-		_, err = conf.GetInstanceServer(server)
+		err := c.addRemoteFromToken(addr, server, token, rawToken.Fingerprint)
 		if err != nil {
-			certificate, err = shared.GetRemoteCertificate(addr, c.global.conf.UserAgent)
-			if err != nil {
+			if api.StatusErrorCheck(err, http.StatusServiceUnavailable) {
```

Comment:
I wonder if we should print the error to stderr too so the user can see which server addresses are being tried and which ones are failing?